functions/Get-DbaDatabaseSnapshot.ps1

```diff
@@ -55,18 +54,20 @@ Returns informations for database snapshots excluding ones that have HR as base
 		[string[]]$SqlServer,
 		[PsCredential]$Credential
 	)
-
-	DynamicParam {
-		if ($SqlServer) {
-			return Get-ParamSqlDatabases -SqlServer $SqlServer[0] -SqlCredential $Credential
+	
+	DynamicParam
+	{
+		if ($SqlServer)
+		{
+			Get-ParamSqlSnapshotsAndDatabases -SqlServer $SqlServer[0] -SqlCredential $Credential
 		}
 	}
-
+	
 	BEGIN
 	{
 		# Convert from RuntimeDefinedParameter object to regular array
 		$databases = $psboundparameters.Databases
-		$exclude = $psboundparameters.Exclude
+		$snapshots = $psboundparameters.Snapshots
 	}
 
 	PROCESS
```

Comment:
foreach ($Instance in $SqlInstance)

will be our variable of choice from now on